.buildkite/scripts/run_models.sh

```diff
@@ -16,12 +16,21 @@ db=$1
 echo `pwd`
 cd integration_tests
 dbt deps
+if [ "$db" = "databricks-sql" ]; then
+dbt seed --vars '{salesforce_marketing_cloud_schema: sfmc_sqlw_tests}' --target "$db" --full-refresh
```

Comment:
Should there be a dbt compile statement after this line, similar to `dbt_fivetran_log` ([link](https://github.com/fivetran/dbt_fivetran_log/blob/main/.buildkite/scripts/run_models.sh#L40C87-L41C75))?

Reply:
A compile statement is not necessary. A compile will already be run as part of the run operation. The compile in Fivetran Log may have been for a particular reason, but I don't imagine we need to include it here.